cmd/consumer/main.go

```diff
@@ -76,14 +78,15 @@ func run(ctx context.Context) error {
 	return closeConsumerGroup(ctx, consumerGroup, cfg.GracefulShutdownTimeout)
 }
 
-func runConsumerGroup(ctx context.Context, cfg *Config) (*kafka.ConsumerGroup, error) {
+func runConsumerGroup(ctx context.Context, cfg *config.Config) (*kafka.ConsumerGroup, error) {
 	log.Event(ctx, "[KAFKA-TEST] Starting ConsumerGroup (messages sent to stdout)", log.INFO, log.Data{"config": cfg})
 	kafka.SetMaxMessageSize(int32(cfg.KafkaMaxBytes))
 
 	// Create ConsumerGroup with channels and config
 	cgChannels := kafka.CreateConsumerGroupChannels(1)
 	cgConfig := &kafka.ConsumerGroupConfig{KafkaVersion: &cfg.KafkaVersion}
-	cg, err := kafka.NewConsumerGroup(ctx, cfg.Brokers, cfg.ConsumedTopic, cfg.ConsumedGroup, cgChannels, cgConfig)
+
+	cg, err := kafka.NewConsumerGroup(ctx, cfg.KafkaAddr, cfg.InstanceStartedTopic, cfg.InstanceStartedGroup, cgChannels, cgConfig)
```

Comment:
The topic whould be `cfg.CategoryDimensionImportTopic`, which is the topic used by the producer in the service (and the one we want to consume by this test consumer)